crates/pathfinder/src/sequencer/reply.rs

```diff
@@ -312,21 +292,17 @@ pub mod transaction {
         #[serde_as(as = "Option<Vec<U256AsDecimalStr>>")]
         #[serde(default)]
         pub constructor_calldata: Option<Vec<U256>>,
```

Comment:
Are you sure this type is `U256`? RPC defines it as a `Felt`.

Reply:
This is the data as output by the sequencer and some of those felts are actually output as long ints. I can add some newtypes that wrap StarkHash but deserialize from long uints, which would help us get rid of this U256.

Reply:
This way you will not see any U256 et al in those structs, yet the deserialization will treat them as long uints.